proposals/NNNN-Keyboard-Enhancements.md

```diff
@@ -0,0 +1,225 @@
+# Keyboard Enhancements
+
+* Proposal: [SDL-NNNN](NNNN-Keyboard-Enhancements.md)
+* Author: [Ashwin Karemore](https://github.com/ashwink11)
+* Status: **Awaiting review**
+* Impacted Platforms: [ Android / iOS / RPC / Core]

```

Comment:
@ashwink11 one last comment - can you please update `Android` to `Java Suite`?  Thanks!

Reply:
Done.